README.md

```diff
@@ -11,12 +11,37 @@ Tekton !
 
 Download the latest binary executable for your operating system:
 
-// Note:- Need to update below links.
-* [Max OS X](README.md)
-* [Linux AMD 64](README.md)
-* [Windows AMD 64](README.md)
+* [Max OS X](https://github.com/tektoncd/cli/releases/download/v0.1.0/tektoncd-cli_0.1.0_Darwin_x86_64.tar.gz)
+
+  ```shell
+  # Get the tar.xz
+  wget https://github.com/tektoncd/cli/releases/download/v0.1.0/tektoncd-cli_0.1.0_Darwin_x86_64.tar.gz
+  # Extract tkn in your PATH (e.g. /usr/local/bin)
+  sudo tar xvzf tektoncd-cli_0.1.0_Darwin_x86_64.tar.gz -C /usr/local/bin tkn
+  ```
+  
+* [Linux AMD 64](https://github.com/tektoncd/cli/releases/download/v0.1.0/tektoncd-cli_0.1.0_Linux_x86_64.tar.gz)
+  
+  ```shell
+  # Get the tar.xz
+  wget https://github.com/tektoncd/cli/releases/download/v0.1.0/tektoncd-cli_0.1.0_Linux_x86_64.tar.gz
+  # Extract tkn in your PATH (e.g. /usr/local/bin)
+  sudo tar xvzf tektoncd-cli_0.1.0_Linux_x86_64.tar.gz -C /usr/local/bin/ tkn
+  ```
+  
+* [Linux ARM 64](https://github.com/tektoncd/cli/releases/download/v0.1.0/tektoncd-cli_0.1.0_Linux_arm64.tar.gz)
+  
+  ```shell
+  # Get the tar.xz
+  wget https://github.com/tektoncd/cli/releases/download/v0.1.0/tektoncd-cli_0.1.0_Linux_arm64.tar.gz
+  # Extract tkn in your PATH (e.g. /usr/local/bin)
+  sudo tar xvzf tektoncd-cli_0.1.0_Linux_arm64.tar.gz -C /usr/local/bin/ tkn
+  ```
+  
+* [Windows AMD 64](https://github.com/tektoncd/cli/releases/download/v0.1.0/tektoncd-cli_0.1.0_Windows_x86_64.tar.gz)
+
+  Extract the archive and add it to your `PATH`.
```

Comment:
is it tested by someone if not we should probably removes it until we are sure it actually works?

Reply:
does windows supports tar.gz by default now? shouldnt it be a zip file or something?